packages/app/src/app/pages/Dashboard/Components/ContributionBranch/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { v2BranchUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { Icon, ListAction, Stack, Text } from '@codesandbox/components';
+import { useAppState } from 'app/overmind';
+import React from 'react';
+import { css } from '@styled-system/css';
+import { useHistory } from 'react-router-dom';
+import { DashboardContributionBranch } from '../../types';
+
+export const ContributionBranch: React.FC<DashboardContributionBranch> = ({
+  branch,
+}) => {
+  const {
+    dashboard: { viewMode },
+  } = useAppState();
+  const { name, project } = branch;
+  const url = v2BranchUrl({ name, project });
+  const history = useHistory();
+
+  const { repository } = project;
+
+  const handleClick = (e: React.MouseEvent | React.KeyboardEvent) => {
+    // TODO: add analytics
+    if (e.metaKey) {
+      window.open(url, '_blank');
+    } else {
+      history.push(url);
+    }
+  };
+
+  if (viewMode === 'grid') {
+    return (
+      <Stack
+        aria-label={`Open branch ${name} from ${repository.name} by ${repository.owner} in the editor`}
+        css={css({
+          cursor: 'pointer', // Revisit cursor.
+          position: 'relative',
+          overflow: 'hidden',
+          borderRadius: '4px',
+          opacity: 0.8,
+          transition: 'opacity ease-in-out',
+          transitionDuration: theme => theme.speeds[4],
+          ':hover, :focus, :focus-within': {
+            // This is not the official transition.
+            opacity: 1,
+          },
+          ':focus-visible': {
+            boxShadow: '0 0 2px 1px rgba(255, 255, 255, 0.4)',
+          },
+        })}
+        direction="vertical"
+        onClick={handleClick}
+        onKeyDown={e => e.key === 'Enter' && handleClick(e)}
+        tabIndex={0}
+        // TODO: refine semantics when/if the context menu gets implemented.
+        role="link"
```

Comment:
👍 

This can probably be an `a` element then. That way we don't have to take meta key presses and `onKeyDown` etc. in account. Or does the context menu automatically make it a button?

Reply:
@tristandubbeld because the action of clicking into these links needs to be tracked, we still need to add a click handler even if it's an anchor. So I'd keep the semantics like this so we get the benefit of not having to override default anchor styles. Wdyt?

Reply:
I think overriding the styles of the anchor is easier / less code than the current implementation. I'm also not sure about the focusability or the registration as an interactive item right now. An `a` needs a `href` to be focusable, but I'm not sure if `role="link"` overrides this and makes it that we don't need an `href`.

Reply:
@tristandubbeld there's a tabIndex right above the `role="link"`, not sure if you saw it.

Reply:
👍